SoftLayer/CLI/virt/create.py

```diff
@@ -272,26 +272,27 @@ def cli(env, **args):
         result = vsi.verify_create_instance(**data)
         total_monthly = 0.0
         total_hourly = 0.0
+        total_preset_monthly = 0.0
+        total_preset_hourly = 0.0
 
         table = formatting.Table(['Item', 'cost'])
         table.align['Item'] = 'r'
         table.align['cost'] = 'r'
 
-        for price in result['prices']:
-            total_monthly += float(price.get('recurringFee', 0.0))
-            total_hourly += float(price.get('hourlyRecurringFee', 0.0))
-            if args.get('billing') == 'hourly':
-                rate = "%.2f" % float(price['hourlyRecurringFee'])
-            elif args.get('billing') == 'monthly':
-                rate = "%.2f" % float(price['recurringFee'])
+        if result['presetId']:
+            ordering_mgr = SoftLayer.OrderingManager(env.client)
+            preset_prices = ordering_mgr.get_preset_prices(result['presetId'])
```

Comment:
If they are ordering a preset, what you should do is get the preset prices, and then add the server and ram items to the `results['prices']` array. It looks like the receipt already has the disk prices so no worries there.  

```python
if result['presetId']:
    ordering_mgr = SoftLayer.OrderingManager(env.client)
    preset_prices = ordering_mgr.get_preset_prices(result['presetId'])
    search_keys = ["guest_core", "ram"]
    for price in preset_prices['prices']:
        if price['item']['itemCategory']['categoryCode'] in search_keys:
            result['prices'].append(price)
```

Reply:
Why the result data doesn't have the core/ram info is a mystery to me, but that was the best I could think of to get that data but NOT get all the other price data that gets returned.

Reply:
Done.